### PR TITLE
Clarify kurl cluster update steps

### PR DESCRIPTION
**Pull request by paigecalvert**
https://deploy-preview-2557--replicated-docs.netlify.app/enterprise/updating-kurl#update-the-kurl-cluster

### CODE REVIEW

File: docs/enterprise/updating-kurl.mdx [hunk 1]
```diff
@@ -31,13 +31,15 @@ The Kubernetes scheduler automatically reschedules Pods to other nodes during ma
 
 To update the kURL cluster in an online environment:
 
+1. Edit the kURL installer spec as desired. For example, update the version of Kubernetes or add, remove, or update add-ons. For more information, see [Creating a kURL Installer](/vendor/packaging-embedded-kubernetes).
+
```
Comment (paigecalvert) [inline docs/enterprise/updating-kurl.mdx:35]: ^ Add first step to update the kurl spec

File: docs/enterprise/updating-kurl.mdx [hunk 2]
```diff
@@ -54,28 +56,23 @@ For air gap installations, you must load images on each node in the cluster befo
 
 To update the kURL cluster in an air gap environment:
 
-1. On each node in the cluster, download the kURL `.tar.gz` bundle provided by your software vendor and extract the contents:
-
-   ```bash
-   tar -xvzf FILENAME.tar.gz
-   ```
-   Replace `FILENAME` with the name of the kURL `.tar.gz` bundle.
```
Comment (paigecalvert) [inline docs/enterprise/updating-kurl.mdx:62]: ^ removed this command (which just shows how to extract the bundles and not actually how to download them) and replaced it with a link to more complete instructions

File: docs/enterprise/updating-kurl.mdx [hunk 3]
```diff
@@ -54,28 +56,30 @@ For air gap installations, you must load images on each node in the cluster befo
 
 To update the kURL cluster in an air gap environment:
 
-1. On each node in the cluster, download the kURL `.tar.gz` bundle provided by your software vendor and extract the contents:
+1. Edit the kURL installer spec as desired. For example, update the version of Kubernetes or add, remove, or update add-ons. For more information, see [Creating a kURL Installer](/vendor/packaging-embedded-kubernetes).
+
```
Comment (paigecalvert) [inline docs/enterprise/updating-kurl.mdx:60]: ^ add the same new step to the air gap procedures

File: docs/enterprise/updating-kurl.mdx [hunk 4]
```diff
@@ -54,28 +56,30 @@ For air gap installations, you must load images on each node in the cluster befo
...
+   Then, extract the contents:
 
    ```bash
    tar -xvzf FILENAME.tar.gz
    ```
-   Replace `FILENAME` with the name of the kURL `.tar.gz` bundle.
-
-   For more information, see [Download the kURL Bundle](/vendor/releases-download-airgap-bundles#installer-bundle) in _Downloading Air Gap Bundles_.
+   Replace `FILENAME` with the name of the downloaded kURL `.tar.gz` air gap bundle.
```
Comment (paigecalvert) [inline docs/enterprise/updating-kurl.mdx:68]: ^ just refactored this step to better show that the command provided only extracts the contents of the downloaded bundle, it doesn't actually download the bundle

**Review by xavpaice — APPROVED** [review]

**Review by 5hubh49 — APPROVED** [review]

**Pull request merged**